### PR TITLE
[Design] DSCO - Tabs component - add native tooltip when tab label overflows

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR updates the DSCO `Tabs` component so that when the Tab label text overflows, a tooltip is displayed on hover.
I consulted with @levadadenys and he recommended setting the `title` attribute when the label text overflows. 7bbf26e2797e39d5bc7decf5049d28f0b8357b26

I added a Storybook example of tabs with overflown label text - on hover you can view tooltip.

https://github.com/user-attachments/assets/1ed88e8e-759e-47cd-a4c8-2ceef959e15d

This was needed in `aichat` levels for the workspace tabs which are shown when a teacher user views student chat history and tests the student-customized chatbot.

https://github.com/user-attachments/assets/e6574907-5f95-49b8-b1d0-049c0d9ed65a

## Links
[jira](https://codedotorg.atlassian.net/browse/LABS-1029)
[Slack thread](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1728320508551979)

## Testing story
Tested locally in an `aichat` level and also ran storybook locally and confirmed tooltip is displayed for `Tabs` example added.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/componentLibrary/tabs/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.3.1]()
+## [0.3.2]()
```
Comment (levadadenys) [inline apps/src/componentLibrary/tabs/CHANGELOG.md]: Please add link to this pr here

**Review by levadadenys — COMMENTED** [review]
LGTM in general, but one thing I think worth discussing before merging.
If we add support of native `title` - we need to check with design team on how we want ho handle the case when tab has both Tooltip and overflown text (see screenshot)

Знімок екрана 2024-10-09 о 11 58 39

**Comment by fisher-alice** [comment]
> LGTM in general, but one thing I think worth discussing before merging. If we add support of native `title` - we need to check with design team on how we want ho handle the case when tab has both Tooltip and overflown text (see screenshot)
> 

Thanks! When I chatted with @markabarrett - we discussed not showing native tooltip when tooltip prop is passed. Thanks for the reminder. Updated. Here's a screencast vid after update:

https://github.com/user-attachments/assets/5456eef9-9de6-46a8-bdc2-6062ce8a68d8

File: apps/src/aichat/views/chatWorkspace.module.scss [hunk 2]
```diff
@@ -25,6 +26,12 @@ $default-spacing: 16px;
 
 .tabsContainer {
   background-color: #eaebeb;
+  padding: $tabs-default-spacing $tabs-default-spacing 0 $tabs-default-spacing;
```
Comment (levadadenys) [inline apps/src/aichat/views/chatWorkspace.module.scss]: Not sure I understand. Do we need updates in this file or are they added here accidentaly?

If we do need this styles, then it's not very critical, but might be better to make a separate PR for it (in case for any reason we'll need to revert current Tabs update for example)

**Review by levadadenys — APPROVED** [review]

File: apps/src/aichat/views/chatWorkspace.module.scss (hunk 2 above)
Comment (fisher-alice) [reply apps/src/aichat/views/chatWorkspace.module.scss]: Oh yes - I reverted so that we keep DSCO updates isolated. Thanks!
